soc/nxp/imxrt/Kconfig.defconfig

```diff
@@ -82,8 +82,7 @@ choice SEGGER_SYSVIEW_SECTION
 endchoice
 
 config MBEDTLS
-	default y if CSPRNG_ENABLED
-	depends on ENTROPY_GENERATOR
+	default y if CSPRNG_AVAILABLE
```

Comment:
I wonder if it wouldn't be better to not drive configuration of MBEDTLS based on the setting of CSPRNG.

When I added this, part of the reason you see these in the NXP parts is that we also had situations where we were trying to select with crypto library to use. I think though that we shouldn't automatically enable MBEDTLS but instead leave that to the subsystems and applications.

Reply:
> I think though that we shouldn't automatically enable MBEDTLS but instead leave that to the subsystems and applications.

I agree. Now that the CI is OK, I can add more commits where I remove stuff like the one you proposed here. For example there is also [this change](https://github.com/zephyrproject-rtos/zephyr/blob/fcbf70f44d3c6d072288f8505bb1335faabdfb94/boards/nordic/nrf5340_audio_dk/Kconfig.defconfig#L63) that I introduced in #79931 which should no more needed now.